src/lib/eZ/FieldType/RichText/SearchField.php

```diff
@@ -72,13 +72,16 @@ private function extractText(DOMNode $node)
     /**
      * Extracts short text content of the given $document.
      *
+     * @internal Only for use by RichText FieldType itself.
+     *
      * @param \DOMDocument $document
      *
      * @return string
      */
-    private function extractShortText(DOMDocument $document)
+    public static function extractShortText(DOMDocument $document)
```

Comment:
```suggestion
    public static function extractShortText(DOMDocument $document): string
```
TBH I think I overlooked it when reviewing ezsystems/ezpublish-kernel#2517. To have a clean implementation here, it should be extracted to a common Service. But not insisting, because it's a port and it's simple. Just sayin' ;)